tools/build_cros_gcc

```diff
@@ -105,19 +108,28 @@
    exit 0
  fi
 
- $0 -c
+ rm -rf \
+ ./$GLIBC \
+ ./$GMP \
+ ./$GCC \
+ ./$BINUTILS \
+ ./linux-$KERNEL_VERSION \
+ ./$MPFR \
+ ./$CLOOG \
+ ./$ISL \
+ ./$MPC
 
- echo $1 $2 $3
+ echo $0 $1 $2 $3
 
- wget https://ftpmirror.gnu.org/gnu/binutils/$BINUTILS.tar.xz
- wget https://ftpmirror.gnu.org/gnu/gcc/$GCC/$GCC.tar.xz
- wget https://www.kernel.org/pub/linux/kernel/v"$MAJOR_VERSION".x/linux-$KERNEL_VERSION.tar.xz
- wget https://ftpmirror.gnu.org/gnu/libc/$GLIBC.tar.xz
- wget https://ftpmirror.gnu.org/gnu/mpfr/$MPFR.tar.xz
- wget https://ftpmirror.gnu.org/gnu/gmp/$GMP.tar.xz
- wget https://ftpmirror.gnu.org/gnu/mpc/$MPC.tar.gz
- wget http://isl.gforge.inria.fr/$ISL.tar.bz2
- wget https://www.bastoul.net/cloog/pages/download/$CLOOG.tar.gz
+ wget -c https://ftpmirror.gnu.org/gnu/binutils/$BINUTILS.tar.xz
+ wget -c https://ftpmirror.gnu.org/gnu/gcc/$GCC/$GCC.tar.xz
+ wget -c https://www.kernel.org/pub/linux/kernel/v"$MAJOR_VERSION".x/linux-$KERNEL_VERSION.tar.xz
+ wget -c https://ftpmirror.gnu.org/gnu/libc/$GLIBC.tar.xz
+ wget -c https://ftpmirror.gnu.org/gnu/mpfr/$MPFR.tar.xz
+ wget -c https://ftpmirror.gnu.org/gnu/gmp/$GMP.tar.xz
+ wget -c https://ftpmirror.gnu.org/gnu/mpc/$MPC.tar.gz
+ wget -c http://isl.gforge.inria.fr/$ISL.tar.bz2
+ wget -c https://www.bastoul.net/cloog/pages/download/$CLOOG.tar.gz
```

Comment:
Why do you need `-c`?

Reply:
If the download fails during the script running, it can complete the download. Otherwise, it overwrites it.

Reply:
What does that mean?  That doesn't make any sense.

Reply:
Have you ever canceled a download halfway through? With the `-c` flag, you can continue downloading where you left off.

Reply:
If the downloads abort, does the script exit gracefully?  This is usually why we need sha256 checksums on files.  Why aren't they in this script? :)

Reply:
It continues. Usually, compressed files don't decompress if they fail. All of these also use HTTPS, which automatically checks the files.

Reply:
Also, with the `-c` option, the downloads don't fail. They just remove the corrupted bytes and go on.

Reply:
HTTPS is just an encrypted transfer protocol.  It has nothing to do with checking the file integrity.  I still think it would be a good idea to include the sha256 checksum hash check.  It would be easy, right?  All the checksums are included in each package already.

Reply:
Except for GCC 8.2.0

Reply:
How do you do that in bash, anyway?

Reply:
Would this work?
```
SUMS="5d20086ecf5752cc7d9134246e9588fa201740d540f7eb84d795b1f7a93bca86  $BINUTILS.tar.xz"
SUMS+="325adf3710ce2229b7eeb9e84d3b539556d093ae860027185e7af8a8b00a750e  $CLOOG.tar.gz"
SUMS+="196c3c04ba2613f893283977e6011b2345d1cd1af9abeac58e916b1aab3e0080  $GCC.tar.xz"
SUMS+="94efeb00e4603c8546209cefb3e1a50a5315c86fa9b078b6fad758e187ce13e9  $GLIBC.tar.xz"
SUMS+="87b565e89a9a684fe4ebeeddb8399dce2599f9c9049854ca8c0dfbdea0e21912  $GMP.tar.xz"
SUMS+="b587e083eb65a8b394e833dea1744f21af3f0e413a448c17536b5549ae42a4c2  $ISL.tar.bz2"
SUMS+="5833cd780fd96614b722040041d8242c102ad7eb956d6f102760bc0faad0a08e  linux-$KERNEL_VERSION.tar.xz"
SUMS+="6985c538143c1208dcb1ac42cedad6ff52e267b47e5f970183a3e75125b43c2e  $MPC.tar.gz"
SUMS+="67874a60826303ee2fb6affc6dc0ddd3e749e9bfcb4c8655e3953d0458a6e16e  $MPFR.tar.xz"
for SUM in $SUMS; do
  if ! echo "$SUM" | shasum -a 256 -c - || { echo "\033[1;31m$SUM checksum mismatch.  Try again. :/\033[0m"; exit 1; }
done
```

Reply:
Not bad.

Reply:
But your shasum for the Linux sources is off.

Reply:
How so?

Reply:
I changed the sources from 3.18.1 to 3.18

Reply:
Also, your `if` statement is off.

Reply:
Your `for` statement won't work, because the SUMS variable has spaces in it. It also doesn't have any newlines.

Reply:
Dude, the `IFS` line makes it work with newlines.

Reply:
Your comments came after I edited.

Reply:
Yes, but your edits are inside the `Load more...` prompt.

Reply:
Are you going to add sha256 sums?